estate/models/estate.py

```diff
@@ -0,0 +1,26 @@
+from odoo import fields, models
+
+class Estate(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
file name should be same as name of the model ex: pos_order.py
refer codebase for reference